dlang.org.ddoc

```diff
@@ -164,11 +174,11 @@ $(DIVID tools,
 )
 _=
 
-PC=<p class="$1">$+</p>
+PC=$(TC p, $1, $+)
 _=
 
-PHOBOS=<a href="http://dlang.org/phobos/std_$1.html#$2">$3</a>
-PHOBOSSRC=$(LINK2 https://github.com/D-Programming-Language/phobos/blob/master/$0, $0)
+PHOBOS=$(SPANC phobos, $(AHTTP dlang.org/phobos/std_$1.html#$2, $(TAIL $3)))
```

Comment:
I guess this should be 
`PHOBOS=$(SPANC phobos, $(AHTTP dlang.org/phobos/std_$1.html#$2, $(TAIL $2)))`